subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CopyErrorIntegrationTest.groovy

```diff
@@ -72,7 +72,7 @@ The following types/formats are supported:
 
```

Comment:
Please use imperative mode in commit titles, "Fix copy related tests" in this particular case.

Reply:
I think that was already imperative, changed it to past tense

Reply:
Reverted this test - left only the formatting (it helped me with keeping track of related tests), can revert if needed

Reply:
I was referring to the title of the original commit, not the file. I don't see the commit anymore, resolving this.